test/utils/nonsipp/check/LoansCheckStatusUtilsSpec.scala

```diff
@@ -0,0 +1,406 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils.nonsipp.check
+
+import models.IdentityType._
+import utils.nonsipp.check.LoansCheckStatusUtils.{checkLoansRecord, checkLoansSection}
+import org.scalatest.OptionValues
+import models._
+import pages.nonsipp.common._
+import models.SponsoringOrConnectedParty.{ConnectedParty, Neither, Sponsoring}
+import org.scalatest.matchers.must.Matchers
+import models.ConditionalYesNo._
+import config.RefinedTypes.Max5000
+import controllers.ControllerBaseSpec
+import pages.nonsipp.loansmadeoroutstanding._
+import models.IdentitySubject.LoanRecipient
+
+class LoansCheckStatusUtilsSpec extends ControllerBaseSpec with Matchers with OptionValues {
```

Comment:
No changes were made to this Spec, I just copied the LOP related code from CheckStatusUtilsSpec and made this its own file.